test/deserializers/params-fields.test.js

```diff
@@ -0,0 +1,10 @@
+const ParamsFieldsDeserializer = require('../../src/deserializers/params-fields');
+
+describe('deserializers > params-fields', () => {
+  it('should split commas separated strings into arrays', () => {
+    expect.assertions(1);
+    const paramsFields = { model: 'field1,field2,field3' };
+    const result = (new ParamsFieldsDeserializer(paramsFields)).perform();
+    expect(result).toStrictEqual({ model: ['field1', 'field2', 'field3'] });
```

Comment:
[STYLE] I like to have an `expectedFoo` and an `actualFoo` variable.
I feel it gives an easier and quick visualization of the input and output.